pilot/pkg/serviceregistry/kube/controller/controller_test.go

```diff
@@ -383,15 +383,27 @@ func TestController_GetPodLocality(t *testing.T) {
 				generateNode("node2", map[string]string{NodeZoneLabel: "zone2"}),
 			},
 			wantAZ: map[*coreV1.Pod]string{
-				pod1: "/zone1",
-				pod2: "/zone2",
+				pod1: "/zone1/",
```

Comment:
was this even correct before? Not sure how the rest of the code handles a missing region...

Reply:
Good question I need to take a look.